container/docker/factory.go

```diff
@@ -172,9 +172,12 @@ func (self *dockerFactory) DebugInfo() map[string][]string {
 	return map[string][]string{}
 }
 
+var (
+	version_regexp_string = "(\\d+)\\.(\\d+)\\.(\\d+)"
+	version_re            = regexp.MustCompile(version_regexp_string)
```

Comment:
What's the point of moving this out of the function? Personally I find it more readable if I can see the regexp right where it's used (and I don't think this should be used outside that function).